packages/manager/modules/models/billing/Bill.class.js

```diff
@@ -0,0 +1,36 @@
+export default class Bill {
+  constructor({
+    billId,
+    category,
+    date,
+    debt,
+    pdfBetaUrl,
+    pdfUrl,
+    priceWithTax,
+    priceWithoutTax,
+    url,
+  }) {
+    Object.assign(this, {
+      billId,
+      category,
+      date,
+      pdfBetaUrl,
+      pdfUrl,
+      priceWithTax,
+      priceWithoutTax,
+      url,
+    });
+
+    this.debt = debt || {
+      status: 'PAID',
+      dueAmount: {
+        value: 0,
+        text: priceWithTax.text.replace(/([0-9]|\.|,)+/g, '0.00'),
```

Comment:
Those are data from the API ? Are you sure it matches all prices with all currencies ? Can a debt be negative ?

Reply:
Properties are from the API. No a debt can't be negative